src/components/Message.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+interface MessageProps {
+  message: string;
+  isSent: boolean;
+}
+
+const Message: React.FC<MessageProps> = ({ message, isSent }) => {
+  // Define styles for sent messages
+  const sentStyles: React.CSSProperties = {
+    maxWidth: "264px",
+    borderTopLeftRadius: "16px",
+    borderBottomRightRadius: "16px",
+    borderBottomLeftRadius: "16px",
+    padding: "12px",
+    background: "#FFDFDF",
+    marginLeft: "auto",
+    wordBreak: "break-word",
+    fontSize: "14px",
+  };
+  // Define styles for received messages
+  const receivedStyles: React.CSSProperties = {
+    maxWidth: "264px",
+    borderTopRightRadius: "16px",
+    borderBottomRightRadius: "16px",
+    borderBottomLeftRadius: "16px",
+    padding: "12px",
+    background: "#DEDEDE4D",
+    marginRight: "auto",
+    wordBreak: "break-word",
+    fontSize: "14px",
+  };
+
+  // Choose styles based on the isSent prop
+  const styles = isSent ? sentStyles : receivedStyles;
+
+  return (
+    <div
+      style={{
+        display: "flex",
+        width: "100%",
+        marginBottom: "10px",
+      }}
+    >
+      <div style={styles}>{message}</div>
```

Comment:
One thing I noticed in the storybook is that the way this renders, we aren't able to put in new lines in the text here: 
![image](https://github.com/user-attachments/assets/3c298834-9b22-4191-9106-3a89f9453b6f)

Something to think about, but also something that can be tackled/fixed later on. I know this PR has been open for a while now.